docs/milo/manual/chapters/40_y_axis_assembly.md

```diff
@@ -188,7 +188,7 @@ Fasten the Y-Axis Motor Mount to the 470mm C-Beam using M5x12mm BHCS.
 Fasten the Y-Axis bearing block to the other end of the 470mm C-Beam using M5x12BHCS
 
 !!! tip
-    The Y-Axis Bearing Block is used as a stopper to install the rails - it will be removed later, so there is no need to threadlock it at this point.
+    The Y-Axis Bearing Block is used as a stopper to install the rails - it will be removed later, so there is no need to threadlock it.
```

Comment:
I think the original text is clear in its intention - don't threadlock it now, but remember to threadlock it later after reinstalling.